### PR TITLE
JAMES-3946 Implement a droplist extension

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

File: server/apps/distributed-app/docs/modules/ROOT/pages/operate/webadmin.adoc [hunk 1]
```diff
@@ -2835,6 +2835,126 @@ Response codes:
 
 * 204: Operation succeeded
 
+== Administrating DropLists
```
Comment (quantranhong1999) [inline server/apps/distributed-app/docs/modules/ROOT/pages/operate/webadmin.adoc:2838]: We should add this in `manage-webadmin.md` too IMO.

But let's do that in another PR.

**Review by quantranhong1999 — APPROVED** [review]

**Review by Arsnael — APPROVED** [review]

File: server/apps/distributed-app/docs/modules/ROOT/pages/operate/webadmin.adoc (hunk 1 above)
Comment (Maxxx873) [reply server/apps/distributed-app/docs/modules/ROOT/pages/operate/webadmin.adoc:2838]: Thank you! I will fix in another PR
